src/app/screens/Home/layout.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { shape, arrayOf, string, func } from 'prop-types';
+import { push } from 'connected-react-router';
+import { useDispatch } from 'react-redux';
+import { UTIconButton, UTLabel, UTTooltip } from '@widergy/energy-ui';
+import i18 from 'i18next';
+
+import { RECORD } from 'constants/routes';
+
+import { ReactComponent as Icon } from '../../assets/record_icon.svg';
+
+import styles from './styles.module.scss';
+
+const Home = ({ valuesToMap, sentenceToDisplay, actualDisplay, buttonRenderer }) => {
+  const dispatch = useDispatch();
+
+  const goToRecordButtonTippyProps = () => ({
+    placement: 'top',
+    interactive: false,
+    arrow: false,
+    boundary: 'viewport',
+    appendTo: document.body
+  });
+
+  const goToRecordButtonRenderer = () => (
+    <UTTooltip
+      content={<span>{i18.t('Calculator:recordButton')}</span>}
+      tippyProps={goToRecordButtonTippyProps()}
+    >
+      <span className={styles.recordButtonContainer}>
+        <UTIconButton className={styles.calculatorRecordButton} onClick={() => dispatch(push(RECORD))}>
+          <Icon />
+        </UTIconButton>
+      </span>
+    </UTTooltip>
+  );
```

Comment:
This is just used once, there's no need for a renderer. You have 2 options:
a) Remove the renderer and move the content directly inside of `<div className={styles.calculatorResultBox}` or
b) Move it to a new component, for example `IconButtonWithTooltip` and then import it and use it here `<IconButtonWithTooltip />`

Reply:
done!